### DISCUSSION TITLE
Nested selectors in sx prop

### DISCUSSION THREAD

**Discussion by fredshevek** [chakra-ui/chakra-ui#2583] [Q&A]
Hi,

I'm migrating a project from theme-ui to Chakra. For most of the part, it's going well but I couldn't make it work nested selectors in `sx` prop. As soon as I remove `import { jsx } from "theme-ui"` they stop working.

Is this a known limitation or does it happen because I'm in a limbo state?

Thanks

**Comment by fredshevek** [comment]
I think I got my mistake. I've been trying to use the `sx` on a regular `div`, not on a Component.

In this case, what would be the best way to get the same outcome?

**Reply by davebrewer** [reply]
Box component spits out a div right? So try and use the box component and attach sx and see if that works maybe?

**Reply by fredshevek** [reply]
That worked, thank you
